runner/logs.go

```diff
@@ -408,6 +409,9 @@ func testAzureLogStream(ctx context.Context, wg *sync.WaitGroup, server *state.S
 	err := azure.SetupLogSubscriber(ctx, wg, globalCollectionOpts, logger, []*state.Server{server}, parsedLogStream)
 	if err != nil {
 		logger.PrintError("ERROR - Could not get logs through Azure Event Hub: %s", err)
+		if strings.HasPrefix(err.Error(), "failed to configure Azure AD JWT provider: failed to refersh token") { // Note that "refersh" is a typo in the Azure SDK logic
```

Comment:
Should we maybe future-proof against them fixing this?

Reply:
Yeah, maybe I'll just skip the last two words in that error message, that way if they fix the typo, it'll still work.